server/constants/oauth_config.go

```diff
@@ -2,7 +2,7 @@ package constants
 
 const (
 	ResponseType        = "Assertion"
-	Scopes              = "vso.code_full vso.work_full"
+	Scopes              = "vso.build_execute vso.code_full vso.release_manage vso.work_full"
```

Comment:
why `build/release` change till phase 2?

Reply:
Already reverted that change